src/components/TasksTable/index.jsx

```diff
@@ -295,7 +295,7 @@ export default class TasksTable extends Component {
     return (
       <Fragment>
         <Toolbar className={classes.toolbar}>
-          <Typography variant="title" className={classes.title}>
+          <Typography variant="title" className={classes.title} id="tableTitle">
```

Comment:
Remove `id`.

Reply:
I add id because the table is rendered as `<table aria-labelledby='tableTitle'>`

Reply:
Ahh, ok makes sense. Thanks for clarifying.